action/protocol/staking/protocol.go

```diff
@@ -409,6 +409,11 @@ func (p *Protocol) CreatePreStates(ctx context.Context, sm protocol.StateManager
 	if err != nil {
 		return err
 	}
+	if blkCtx.BlockHeight == g.ToBeEnabledBlockHeight {
+		if err := v.(*ViewData).contractsStake.FlushBuckets(sm); err != nil {
```

Comment:
the view stored in the trie will miss the changes in the current block, which may lead to incorrect data being loaded at startup